packages/aws-cdk-lib/aws-apigateway/lib/restapi.ts

```diff
@@ -454,6 +464,31 @@ export abstract class RestApiBase extends Resource implements IRestApi {
     });
   }
 
+  /**
+   * Add a resource policy that only allows API execution from a VPC Endpoint to create a private API.
+   *
+   * @param vpcEndpoint the interface VPC endpoint to grant access to
+   */
+  public grantInvoke(vpcEndpoint: ec2.IVpcEndpoint): void {
```

Comment:
This name `invoke` sounds pretty generic and people may be confused by the name and thought this is some generic grant method to invoke API, but the actual usage is only for VPC endpoint.

Reply:
How about the following method names?
I'm not very proficient in English, so I would appreciate it if you could suggest an appropriate name.

- grantVpcEndpointOnlyInvoke
- allowInvokeFromVpcEndpointOnly

Reply:
I'd just be specific to something like `grantInvokeToVpcEndpoint`.

Reply:
Thanks! I've updated to use `grantInvokeToVpcEndpoint`.